freeipa/src/main/java/com/sequenceiq/freeipa/service/image/FreeIpaImageFilterSettings.java

```diff
@@ -0,0 +1,64 @@
+package com.sequenceiq.freeipa.service.image;
+
+public class FreeIpaImageFilterSettings {
+
+    private String imageId;
```

Comment:
I would name this to currentImageId, as otherwise this could indicate that we are looking for the image with this id